rebar.config

```diff
@@ -1,11 +1,11 @@
 % -*- mode: erlang -*-
 {erl_opts, [debug_info]}.
 
-{port_specs, 
- [{".*-linux", "priv/statfs.so", 
```

Comment:
I haven't checked what this was excluding, but our change would basically remove all restrictions? (as in, Windows, VxWorks, etc?)